ui/src/components/TaskRunsCard/index.jsx

```diff
@@ -247,8 +262,50 @@ export default class TaskRunsCard extends Component {
     };
   }
 
+  renderArtifactRow({ artifact }) {
+    const { classes } = this.props;
+    const { name } = artifact;
+    const {
+      icon: Icon,
+      isLogFile,
+      url,
+      handleArtifactClick,
+    } = this.getArtifactInfo(artifact);
+
+    return (
+      <TableRow
+        key={name}
```

Comment:
If I'm reading this correctly, the `key` value is a lot shorter. Did you intend to change this key from `run-artifact-${run.taskId}-${run.runId}-${artifact.name}` to `${artifact.name}`. Also I don't know what `key` does.

Reply:
It's a React thing -- as long as each element in this array has a different key, everything's fine.  So it didn't need to be so long.

Reply:
https://reactjs.org/docs/lists-and-keys.html#keys